packages/@okta/vuepress-site/docs/reference/token-hook/index.md

```diff
@@ -23,7 +23,7 @@ This type of inline hook is triggered when OAuth 2.0 and OpenID Connect (OIDC) t
 
 This functionality can be used to add data that is sensitive, calculated at runtime, or complexly-structured and not appropriate for storing in Okta user profiles. Data added this way is never logged or stored by Okta. As an example, tokens minted for a medical app could be augmented with confidential patient data provided by your external service and not stored in Okta.
 
-In addition to adding custom claims, you can modify or remove an existing custom claim or an OIDC standard profile claim. You can also update how long an access token or an ID token is valid.
+In addition to adding custom claims, you can modify or remove an existing custom claim or an OIDC standard profile claim. You can update how long an access token or an ID token is valid. You can also refresh the access token using a unique refresh token ID sent in the request. That ID is stored and when the refresh token request is sent, the unique refresh token ID is matched against what is stored before issuing the refreshed access token.
```

Comment:
you can remove this doc change here.

Reply:
next to ###scopes you can add 
### data.refresh_token
Provides the refresh token ID, you could store the ID on the first access token request. When doing refresh token grant, its Id will match what was stored which should allow you to connect the two requests.

| Property | Description                   | Data Type                    |
|----------|-------------------------------|------------------------------|
| jti   | Refresh token ID | String     |

Reply:
@venkatviswanathan-okta please check if this text needs correction.